timetracker/entries/models.py

```diff
@@ -2,24 +2,40 @@
 
 from django.utils import timezone
 
+# Create your models here.
+class Client(models.Model):
+    name = models.CharField(max_length=200)
 
+    def __str__(self):
+        return '<Client: {}>'.format(self.name)
+
+    
 class Project(models.Model):
-    client = models.CharField(max_length=200)
     name = models.CharField(max_length=200)
+    client = models.ForeignKey(Client)
```

Comment:
Little heads up: it's safer to put the model name in quotes in `ForeignKey` and `ManyToMany` field definitions.